docs/app.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import flask
+import flask_login
+
+app = flask.Flask(__name__, static_url_path="/", static_folder="_build/html/")
+app.secret_key = "super secret string"  # Change this!
+
+login_manager = flask_login.LoginManager()
+login_manager.init_app(app)
+
+users = {"pytoda-team@ibm": {"password": os.environ["PYTODA_S3_SECRET_KEY"]}}
```

Comment:
replace `pytoda-team@ibm` with `paccmann-team`, also the env variable was a copy/paste I forgot to change, simply name it `PYTODA_DOCS_PASSWORD`